kaspresso/src/main/kotlin/com/kaspersky/kaspresso/runner/KaspressoRunner.kt

```diff
@@ -0,0 +1,24 @@
+package com.kaspersky.kaspresso.runner
+
+import android.os.Bundle
+import androidx.test.runner.AndroidJUnitRunner
+import com.kaspersky.kaspresso.runner.listener.KaspressoRunNotifier
+import com.kaspersky.kaspresso.runner.listener.KaspressoRunNotifierImpl
+import com.kaspersky.kaspresso.runner.listener.SpyRunListener
+
+@Suppress("UNUSED")
+class KaspressoRunner : AndroidJUnitRunner(), KaspressoRunNotifier by KaspressoRunNotifierImpl() {
+
+    override fun onCreate(arguments: Bundle) {
+        arguments.putArgs("listener", SpyRunListener::class.java.name)
```

Comment:
Simply sets the `SpyRunListener ` to JUnit. This listener notifies us back on the tests lifecycle events, so we can notify other our custom listeners.